docs/source/release/0.59.0-notes.rst

```diff
@@ -456,7 +456,7 @@ Pull-Requests:
 * PR `#9149 <https://github.com/numba/numba/pull/9149>`_: update release checklist post 0.58.0rc1 (`esc <https://github.com/esc>`_)
 * PR `#9154 <https://github.com/numba/numba/pull/9154>`_: Add support for np.unwrap() (`KrisMinchev <https://github.com/KrisMinchev>`_)
 * PR `#9155 <https://github.com/numba/numba/pull/9155>`_: Remove unused test.cmd (`sklam <https://github.com/sklam>`_)
-* PR `#9168 <https://github.com/numba/numba/pull/9168>`_: fix the `get_template_info` method in `overload_method` template (`sklam <https://github.com/sklam>`_)
+* PR `#9168 <https://github.com/numba/numba/pull/9168>`_: fix the `get_template_info` method in `overload_method` template (`dlee992 <https://github.com/dlee992>`_ `sklam <https://github.com/sklam>`_)
```

Comment:
The list of all the authors at the end of the release notes for 0.59.0 needs `dlee992` adding too now this entry is fixed?

Reply:
good catch